examples/graphql-client-query-endpoint/graphql_client_query_endpoint.md

```diff
@@ -1,13 +1,11 @@
 # GraphQL client - Query GraphQL endpoint
 
-The GraphQL Client can be used to connect and interact with a GraphQL server. It can be used to do `Query` and `Mutation` operations on a GraphQL service.
+The GraphQL client can be used to connect and interact with a GraphQL server.  A GraphQL client is created by passing the URL of a GraphQL service endpoint. To execute a GraphQL operation the `execute` method is used. This method requires the GraphQL `document` as a required argument and optionally takes a map of `variables` and an `operationName` as arguments in case the document contains any variables or contains more than one operation. Use the GraphQL client when you want to execute `Query` and `Mutation` operations on a GraphQL service.
```

Comment:
```suggestion
The GraphQL client can be used to connect and interact with a GraphQL server.  A GraphQL client is created by passing the URL of a GraphQL service endpoint. The `execute` method is used to execute a GraphQL operation. This method requires the GraphQL `document` as a required argument and takes a map of `variables` and an `operationName` as arguments optionally in case the document contains any variables or contains more than one operation. Use the GraphQL client to execute the `Query` and `Mutation` operations on a GraphQL service.
```

Reply:
Updated